src/experiments/preferences/schema.json

```diff
@@ -113,6 +129,50 @@
         ],
         "async": true
       },
+      {
+        "name": "setBoolPref",
+        "type": "function",
+        "description": "Sets the value of a boolean preference",
+        "parameters": [
+          {
+            "type": "string",
+            "name": "name"
```

Comment:
For all the pref set APIs, add an enum that specifies the prefs that are allowed to be set.

```
enum: ["doh-rollout.disable-heuristics", ...],
```

Reply:
@mixedpuppy Added! Note that the prefernces set via the API are :
- `doh-rollout.doorhanger-shown`
- `doh-rollout.previous.trr.mode`
- `doh-rollout.doorhanger-decision`
- `doh-rollout.balrog-migration`
- `doh-rollout.self-enabled`
- `doh-rollout.disable-heuristics`
- `network.trr.mode`